third_party/ibis/ibis_teradata/client.py

```diff
@@ -66,7 +66,7 @@ class TeradataClient(SQLClient):
     table_class = TeradataTable
     dialect = compiler.TeradataDialect
 
-    def __init__(self, host, user_name, password, port=1025, use_no_lock_tables=False):
+    def __init__(self, host, user_name, password, port=1025, logmech='TD2', use_no_lock_tables=False):
         """Construct a TeradataClient.
 
         Parameters
```

Comment:
Nit: Can you add 'logmech' in the function documentation as a parameter?

Reply:
complete